relayer/relayer/src/handlers/azero.rs

```diff
@@ -107,6 +107,17 @@ impl AlephZeroEventHandler {
                 let address = eth_contract_address.parse::<Address>()?;
                 let contract = Most::new(address, eth_signed_connection.clone());
 
+                if past_request(
+                    &contract,
+                    committee_id.into(),
+                    eth_signed_connection.address(),
+                )
+                .await?
+                {
+                    info!("Guardian signature for {request_hash:?} not needed - request from a past committee");
```

Comment:
Strictly formally -- it's not necessarily from "past committee", it's from a committee we are not part of. 

In fact I would consider changing the name `past_request` to `not_in_committee`.

Reply:
True, will rename